packages/api/src/jobs/station_sync/updateTimeZoneOffsets.js

```diff
@@ -0,0 +1,21 @@
+const { Client } = require('@googlemaps/google-maps-services-js');
+
+const client = new Client({});
+
+async function mapTimeZoneOffsetsToFarms(knex) {
+  const farmsWithLatLng = await knex('farm').select('*').whereNotNull('grid_points');
+  for (const farm of farmsWithLatLng) {
+    const timeZone = await client.timezone({
```

Comment:
Please check [here](https://developers.google.com/maps/documentation/timezone/requests-timezone#TimeZoneStatus) to see if any error response handling is needed.

Reply:
I've added error handling and rate limiting checks